libraries/chain/controller.cpp

```diff
@@ -380,6 +380,12 @@ struct controller_impl {
       } catch ( std::exception& e ) {
          wlog( "std::exception: ${details}", ("details", e.what()) );
       } catch ( ... ) {
+         auto expPtr = std::current_exception();
+         try {
+           if(expPtr) std::rethrow_exception(expPtr);
+         } catch(const std::exception& e) {
+           wlog( "${details}", ("details", e.what()) );
+         }
```

Comment:
You can remove all this, it is not needed because `std::exception` handling was added above since this patch was created.